CHANGELOG.md

```diff
@@ -7,9 +7,23 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased](https://github.com/pusher/push-notifications-swift/compare/0.10.10...HEAD)
 
+### Added
+
+- Retry strategy with exponential backoff.
+
+### Removed
+
+- NetworkResponse.
```

Comment:
Isn't this internal? I think the CHANGELOG should be more about visible changes.

Or a summarized version of what happened: e.g. "Internal refactoring of the Network layer"